documentation/docs/pages/contributing/index.md

```diff
@@ -0,0 +1,40 @@
+
+We also manage most of our development openly on GitHub, we track ESM1.6 work on a [GitHub project board here](https://github.com/orgs/ACCESS-NRI/projects/14). If you'd like to have your issue considered for esm1.6 development, please [open an issue](https://github.com/ACCESS-NRI/access-esm1.6-configs/issues/new).
+
+### Report a bug or make a suggestion
+
+Technical issues related to esm1.6 are best posted to [github.com/ACCESS-NRI/access-esm1.6-configs/issues](https://github.com/ACCESS-NRI/access-esm1.6-configs/issues/new). If you would like to discuss the issue first, feel free to post it on the [ACCESS-HIVE forum](https://forum.access-hive.org.au/c/cmip7/cmip7-development/217).
+
+## ACCESS-esm1.6-configs documentation contributions welcome 🙏
+This documentation is a work in progress, and we welcome any contributions, including corrections and suggestions.
+
+All contributions are welcome but we would particularly appreciate text suggestions (below). Contributions can be made in a few ways:
+
+### Quick contributions 
+!!! tip
+    This method has the advantage is that it's *very quick* (<1 minute!). The caveat is that, unless you have write access to the `access-esm1.6-configs` repository, you will not be able to preview the changes rendered into a website or create whole new pages. <br>
+
+The simplest and fastest way to make a change to an _existing_ page is to click the edit "pencil" on the top-right corner. This will go to the relevant GitHub markdown file and clicking the top-right pencil again on GitHub will allow you to edit the file. Once complete, click `Commit changes...`. There are then _two_ possibilities, depending on whether you have  write access to [`access-esm1.6-configs`](https://github.com/ACCESS-NRI/access-esm1.6-configs): 
+
+1.  **No write access** (e.g. you are not part of the `ACCESS-NRI` GitHub organisation): this will prompt you to make a fork and then a pull request (less than 1 minute!). 
+1.  **You have write access**: please commit changes on a new branch and then use a pull request (this relates to the next option). 
+
+### Larger contributions (online PR-previews)
+!!! tip
+    This method allows you create whole new pages, and to preview the changes rendered into a website. It does not require you to install any software, but is **only available for people with write access to [`access-esm1.6-configs`](https://github.com/ACCESS-NRI/access-esm1.6-configs)**.<br>
```

Comment:
Does that mean we should have a "how to get write access" section somewhere? In the config docs or in the README?

Reply:
Can leave it as-is for now.